nextflow_schema.json

```diff
@@ -135,6 +135,11 @@
                     "help_text": "Specifies the tool(s) to use. Available are: `syfpeithi`, `mhcflurry`, `mhcnuggets-class-1`, `mhcnuggets-class-2`. Can be combined in a list separated by comma.",
                     "description": "Specifies the prediction tool(s) to use."
                 },
+                "tool_thresholds": {
+                    "type": "string",
+                    "description": "Specifies the binder threshold for each given tool in JSON format.",
```

Comment:
I think we should make clear here that it should be provided as a file. What do you think?

Reply:
Makes sense! Something like: "Specifies the binder threshold for each given tool in aJSON formatted file."?
We could also include an example in the help section on the next line => {"MyTool:"MyThreshold}. What do you think?

Reply:
How about? `Specifies tool-specific binder thresholds in a JSON file. This can be used to override the given default binder threshold values.`

An example sounds like a good idea 👍